README.md

```diff
@@ -4,43 +4,29 @@ The specification and reference model for the *Symbol Graph* File Format.
-
-```swift
-public struct MyStruct {
-  public var x: Int
-}
-```
-
-There are two nodes in this module's graph: the structure `MyStruct` and its property, `x`:
+By modeling different kinds of relationships, SymbolKit can provide rich data to power documentation, answering interesting questions, such as:
 
-![A graph with 2 nodes: "Struct Node" MyStruct and "Instance Property Node" x](Sources/SymbolKit/SymbolKit.docc/Resources/twonodes@2x.png)
+- *Which types conform to this protocol?*
+- *What is the class hierarchy rooted at this class?*
+- *Which protocol provides a requirement called `count`?*
+- *Which types customize this protocol requirement?*
 
-`x` is related to `MyStruct`: it is a *member* of `MyStruct`. SymbolKit represents *relationships* as directed edges in the graph:
+In addition, graph representations of data also present opportunities for visualizations in documentation, illustrating the structure or hierarchy of a module.
 
-![Node x has a directed edge with text "memberof" to Node MyStruct](Sources/SymbolKit/SymbolKit.docc/Resources/member@2x.png)
+## Getting Started Using Markup
 
-The *source* of an edge points to its *target*. You can read this edge as *`x` is a member of `MyStruct`*. Every edge is qualified by some kind of relationship; in this case, the kind is membership. There can be many kinds of relationships, even multiple relationships between the same two nodes. Here's another example, adding a Swift protocol to the mix:
+In your `Package.swift` Swift Package Manager manifest, add the following dependency to your `dependencies` argument:
 
 ```swift
-public protocol P {}
-
-public struct MyStruct: P {
-  public var x: Int
-}
+.package(url: "https://github.com/apple/swift-docc-symbolkit.git", .branch("main")),
 ```
 
-Now we've added a new node for the protocol `P`, and a new conformance relationship between `MyStruct` and `P`:
+Add the dependency to any targets you've declared in your manifest:
 
-![Node x has a directed edge with text "memberof" to Node MyStruct, Node MyStruct has a directed edge with text "conformsTo" to "Protocol Node" P](Sources/SymbolKit/SymbolKit.docc/Resources/conforms@2x.png)
-
-By modeling different kinds of relationships, SymbolKit can provide rich data to power documentation, answering interesting questions, such as:
-
-- *Which types conform to this protocol?*
-- *What is the class hierarchy rooted at this class?*
-- *Which protocol provides a requirement called `count`?*
-- *Which types customize this protocol requirement?*
+```swift
+.target(name: "MyTarget", dependencies: ["SymbolKit"]),
+```
 
-In addition, graph representations of data also present opportunities for visualizations in documentation, illustrating the structure or hierarchy of a module.
+Please see Swift `SymbolKit`'s [documentation site](https://apple.github.io/swift-docc-symbolkit/documentation/symbolkit/) for more detailed information about the library.
```

Comment:
It looks like we generally refer to SymbolKit without code voice in the existing README.

```suggestion
Please see SymbolKit's [documentation site](https://apple.github.io/swift-docc-symbolkit/documentation/symbolkit/) for more detailed information about the library.
```

Reply:
What do you think of moving this line up directly after:

> In addition, graph representations of data also present opportunities for visualizations in documentation, illustrating the structure or hierarchy of a module.

And before 

> Getting Started Using SymbolKit

?